src/main/java/se/diabol/jenkins/pipeline/DeliveryPipelineView.java

```diff
@@ -117,13 +112,20 @@
 
     private transient String error;
 
+    private static Random generator = new Random();
+    private String id = null;
+    private String viewUrl = null;
+    private boolean isPortletView = false;
```

Comment:
Thinking if it is possible / reasonable to extract the portlet specific fields to an encapsulating PortletConfiguration object, so only one field would be specific to this feature in this class. What do you think?